app/Controllers/commentController.js

```diff
@@ -41,6 +39,27 @@ exports.addCommentToArticle = async (req, res, next) => {
     console.log(newComment)
     res.status(201).send(newComment)
   } catch (error) {
-    next(error)
+    if (error.status === 404) {
+      return res.status(404).send({ message: 'Article not found.' })
+    } else {
+      return res.status(400).send({ message: error.message })
+    }
+  }
+}
+
+exports.deleteCommentByID = async (req, res, next) => {
+  try {
+    const { comment_id } = req.params
+    isValidNumber(comment_id)
+
+    await checkExists('comments', 'comment_id', comment_id)
+    await deleteComment(comment_id)
+    res.status(204).send()
+  } catch (error) {
+    if (error.status === 404) {
+      return res.status(404).send({ message: 'Resource not found' })
+    } else {
+      return res.status(400).send({ message: error.message })
```

Comment:
this error message should be caught in the error handling middleware in your app.  You may have had feedback about this before.

Reply:
@nickkagame Yes, Hannah gave the same feedback. I spent my whole morning fixing the error handling. I was going to ask you to have a call about it, but after her feedback, I understand. I know where to look to find the error logins. Before, I was basically using my tests to catch the error and handle it.

Now, the next() makes sense. Hahaha, Thanks!